packages/core/src/components/dropdown/dropdown-vars.scss

```diff
@@ -10,6 +10,9 @@
   --tds-dropdown-helper-color: var(--tds-grey-700);
   --tds-dropdown-placeholder-color: var(--tds-grey-700);
   --tds-dropdown-disabled-color: var(--tds-grey-400);
+  --tds-dropdown-scrollbar-background: var(--tds-grey-100);
+  --tds-dropdown-scrollbar-thumb-color: var(--tds-grey-500);
```

Comment:
This should be:
`--your-css-variable-name: rgba(0, 0, 0 / 38%);`
And no hover color is needed, merely the thickening of the scrollbar when hovering.

Reply:
Fixed